libibumad/umad.h

```diff
@@ -39,6 +39,7 @@
 #include <stdlib.h>
 #include <arpa/inet.h>
 #include <linux/types.h> /* __be16, __be32 and __be64 */
+#include <infiniband/verbs.h> /* union ibv_gid */
```

Comment:
I have mixed feelings about this. Up to now, umad was not dependent on verbs and on some management machines, verbs was not installed. Now both are in rdma-core so they're together.

If this is to be done, shouldn't there be configure check for this ?